diff --git a/src/csv2rdf/source.clj b/src/csv2rdf/source.clj
index 020df2c..1fa1034 100644
--- a/src/csv2rdf/source.clj
+++ b/src/csv2rdf/source.clj
@@ -58,7 +58,7 @@
 
 (defmulti request-uri-input-stream (fn [^URI uri] (keyword (.getScheme uri))))
 
-(defmethod request-uri-input-stream :http [uri]
+(defmethod request-uri-input-stream :default [uri]
   (let [{:keys [status headers body] :as response} (http/get-uri uri)]
     (if (http/is-not-found-response? response)
       (throw (ex-info